Updates based on review of API documentaton (#276)
redhat-certification/chart-verifier: docs/helm-chart-api.md

# Chart Verifier go language API

IMPORTANT:
- The API was first published in chart verifier release 1.8.0
- The API in release 1.8.0 is not finalized and is subject to change.
- The API is also available in development release 0.1.0.

## Overview

> The API is written using the go language and was created to enable services to run the chart verifier and obtain a report. 

Author: Done.

> @mmulholla Thanks for the changes. I have given some more suggestions. PTAL.
> 
> Also, I see that a good portion of my language suggestions on the use of active voice for descriptions has been ignored. Any particular reason why these language suggestions are ignored or not considered? Please clarify.

I have updated based on those comments, sorry I missed them, hopefully all good now.

}
```

### Description of each function of the APIVerifier interface

- NewReport: Creates a new ```Report```.
  

> - GetContent: Gets the report as a string in either json or yaml format. ReportFormat values are defined and available in the report package:

Reviewer: ```suggestion
- GetContent: Gets the report as a string in either the JSON or YAML format. ReportFormat values are defined and available in the report package:
```

@mmulholla Please incorporate this feedback for all similar instances.

> # Example:

Reviewer: @mmulholla Why is this still pending?

Note: 

> - The following example does not include error checking code for clarity purposes.

Author: @Srivaralakshmi, one thing I have not done is put the api functions into tables. Could not get comfortable with how the table would look and what headings to use. I think it is OK as is for now.

> - For full use of the chart-verifier see:

Reviewer: @mmulholla yes, please incorporate the suggestion

)
```

> 2. Set a profile.vendortype of redhat, un-enable the chart testing check, and run verify:

Reviewer: ```suggestion
2. Set the `profile.vendortype` value to redhat, un-enable the chart testing check, and run verify:
```

Author: done.
